store/src/main/java/com/dropbox/android/external/store4/StoreResponse.kt

```diff
@@ -38,6 +38,11 @@ sealed class StoreResponse<out T> {
      */
     data class Data<T>(val value: T, override val origin: ResponseOrigin) : StoreResponse<T>()
 
+    /**
+     * No new data event dispatched by a Pipeline
```

Comment:
lets document here what this will be followed up with as it won't be very clear.

also, `Fetcher` instead of `Pipeline` ?

Reply:
linking the issue might also make sense for them to easily get the whole context.